chainer/links/loss/crf1d.py

```diff
@@ -1,6 +1,7 @@
 from chainer.functions.loss import crf1d
 from chainer import link
 from chainer import variable
+from chainer import initializers
```

Comment:
A nitpick but could place this import above `from chainer import links`, to sort it alphabetically?

Reply:
Ah, sorry to forget linting...:pray: